crates/ruff_linter/src/rules/pydocstyle/rules/indent.rs

```diff
@@ -179,8 +179,8 @@ pub(crate) fn indent(checker: &Checker, docstring: &Docstring) {
         return;
     }
 
-    let mut has_seen_tab = docstring.indentation.contains('\t');
-    let docstring_indent_size = docstring.indentation.chars().count();
+    let mut has_seen_tab = docstring.indentation().contains('\t');
```

Comment:
Nit: Consider storing the `indentation` (and maybe calling the method `to_indentation` because computing the line start isn't free.